test/Chainweb/Test/Utils.hs

```diff
@@ -1064,10 +1077,12 @@ setBootstrapPeerInfo =
 
 
 host :: Hostname
-host = unsafeHostnameFromText "::1"
+-- host = unsafeHostnameFromText "::1"
+host = unsafeHostnameFromText "localhost"
 
 interface :: W.HostPreference
-interface = "::1"
+-- interface = "::1"
+interface = "127.0.0.1"
```

Comment:
ipv6 -> ipv4?

Reply:
It seems that the tests were more flaky with IPv6. Not sure why.

Reply:
Sounds like a `network` issue.